src/t8_cmesh/t8_cmesh_readmshfile.cxx

```diff
@@ -1415,6 +1411,20 @@ t8_cmesh_msh_file_4_read_eles (t8_cmesh_t cmesh, FILE *fp,
                                   edge_geometries, 
                                   24 * sizeof(int), 
                                   0);
+          
+          /* Now we set the tree geometry according to the tree linkage status. */
+          char geom_name[BUFSIZ];
```

Comment:
```suggestion
          const char *geom_name;
```

Reply:
I do not think that the copying is necessary here.
This should be tested, though.

Reply:
I applied and tested the suggestion